tests: Meiryo is not installed by default on Windows 10

<!--  As an open source project with a dedicated but small maintainer team, it can sometimes take a long time for issues to be addressed so please be patient and we will get back to you as soon as we can.
-->

### Preflight Checklist
<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * <!-- (output of `node_modules/.bin/electron --version`) e.g. 4.0.3 --> 6.0.0-beta.12
* **Operating System:**
  * <!-- (Platform and Version) e.g. macOS 10.13.6 / Windows 10 (1803) / Ubuntu 18.04 x64 --> Windows 10 (1903).
* **Last Known Working Electron version:**
  * <!-- (if applicable) e.g. 3.1.0 -->n/a

### Expected Behavior
<!-- A clear and concise description of what you expected to happen. -->
All font fallback tests pass.

### Actual Behavior
<!-- A clear and concise description of what actually happened. -->
Font fall back tests fail with this in the log:

    not ok 1191 font fallback should fall back to Japanese font for sans-serif Japanese script",
       AssertionError: expected 'Yu Gothic' to equal 'Meiryo'",
           at Context.<anonymous> (C:\\ci\\electron\\spec\\chromium-spec.js:1555:36)",

The Meiryo font does not appear to be installed on any of the Windows 10 machine that I have access to, and [Microsoft's typography page indicates that it's no longer shipped.](https://docs.microsoft.com/en-us/typography/font-list/meiryo)

### To Reproduce
<!--
Your best chance of getting this bug looked at quickly is to provide a REPOSITORY that can be cloned and run.
-->
Run the tests with `-g "font fallback"` or similar on a Windows 10 system.

<!--
You can fork electron-quick-start (https://github.com/electron/electron-quick-start) and include a link to the branch with your changes.
-->

<!--
If you provide a URL, please list the commands required to clone/setup/run your repo e.g.
```sh
$ git clone $YOUR_URL -b $BRANCH
$ npm install
$ npm start || electron .
```
-->

### Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->
n/a

### Additional Information
<!-- Add any other context about the problem here. -->
n/a

--- a/docs/development/testing.md
+++ b/docs/development/testing.md
@@ -42,3 +42,9 @@ you would like to run. As an example: If you want to run only IPC tests, you
 would run `npm run test -- -g ipc`.
 
 [standard-addons]: https://standardjs.com/#are-there-text-editor-plugins
+
+### Testing on Windows 10 devices
+[Some Windows 10 devices](https://docs.microsoft.com/en-us/typography/fonts/windows_10_font_list) do not ship with the Meiryo font installed, which may cause a font fallback test to fail. To install Meiryo:
+1. Push the Windows key and search for _Manage optional features_.
+1. Click _Add a feature_.
+1. Select _Japanese Supplemental Fonts_ and click _Install_.

